server/config.go

```diff
@@ -34,5 +34,11 @@ func DefaultConfig() *Config {
 		JSONRPCAddr: &net.TCPAddr{IP: net.ParseIP("127.0.0.1"), Port: DefaultJSONRPCPort},
 		GRPCAddr:    &net.TCPAddr{IP: net.ParseIP("127.0.0.1"), Port: DefaultGRPCPort},
 		Network:     network.DefaultConfig(),
+		Metrics:     &Telemetry{PrometheusAddr: nil},
 	}
 }
+
+// Telemetry holds the config details for metric services.
```

Comment:
Nitpick warning: Please don't add punctuation to comments

Reply:
Done, Commit [link](https://github.com/0xPolygon/polygon-sdk/pull/216/commits/40e745fb9520b6e8aaae856a1f0a978803abe2ec)